torch_ipex/csrc/jit/register_dnnl_jit_ops.cpp

```diff
@@ -60,7 +60,7 @@ RegisterOperators op({
         return [] (Stack& stack) {
           at::Tensor input;
           pop(stack, input);
-          auto result = dnnl_relu_(input);
+          auto result = AtenIpexCPUDev::dil_relu_(input);
```

Comment:
What's the behavior if the input tensor is non-contiguous?

Reply:
Solved.